activitystream/hide-statuspanel.css

```diff
@@ -0,0 +1,17 @@
+/*
```

Comment:
The status panel isn't part of the activity stream, right? I don't think it fits in any of the current categories. I'm tempted to make a new folder "misc" for this.

Reply:
moved to misc/hide-statuspanel.css